source/Error/ConsoleHandler.ts

```diff
@@ -13,4 +13,4 @@ module Magic.Error {
 			console.log(message.toString())
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
And if so, and if these files didn't already have newlines at the end, where do the files come from?